scripts/dev/gen-demo-loop-kernel.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Copyright 2021 UT-Battelle, LLC and other Celeritas Developers.
+# See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+"""
+Tool to generate demo loop kernel implementations on the fly.
```

Comment:
Are you planning on hooking this up somewhere in the CMakeLists.txt file so that these are autogenerated at config-time (similar to what we do for the interactors)?

Reply:
I am now! Thanks for pointing that out.